x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.test.tsx

```diff
@@ -848,7 +848,7 @@ describe('When on the package policy create page', () => {
 
         expect(sendCreateAgentPolicy).toHaveBeenCalledWith(
           expect.objectContaining({
-            monitoring_enabled: [],
+            monitoring_enabled: ['logs', 'metrics', 'traces'],
```

Comment:
Traces were recently added in this pr: https://github.com/elastic/kibana/pull/189908
And the logic takes all values: https://github.com/elastic/kibana/blob/02a3992edb8021b850e9c66c72f4f5da0d988b82/x-pack/plugins/fleet/common/services/generate_new_agent_policy.ts#L22
I can remove `traces` if we don't want it in agentless.

Reply:
Since I am not familiar with that I am not sure how it's going to affect the user.
I think it's best to have only `logs` and `metrics` which were both tested and we might add `traces` in the future.

Reply:
Traces may help with [[Cloud Security] [Agentless] Include X-Request-ID header in all Agentless API requests and logs](https://github.com/elastic/kibana/issues/188692)

Reply:
> Traces may help with [[Cloud Security] [Agentless] Include X-Request-ID header in all Agentless API requests and logs](https://github.com/elastic/kibana/issues/188692)

`logs` is mainly what we need for the correlation, looking at what [traces](https://www.elastic.co/guide/en/observability/current/apm-traces.html) offers, it can help users detect issues with the communication with the agentless API It could be useful during the agentless beta, however, exposing internal services to the users such as the agentless endpoints and error status is something we had not initially planned. 

So I tend to agree with @amirbenun to enable only `logs` and `metrics` for now and evaluate if we can enable traces in the future

Reply:
Updated with only logs and metrics as requested.